### PR TITLE
Password for social login

### PR DESCRIPTION
**Pull request by danthegoodman1**
See #242

### CODE REVIEW

**Review by vipulgupta2048 — COMMENTED** [review]
@danthegoodman1 Need a better commit message, we all love good messages for our history

**Review by pdcastro — APPROVED** [review]

File: pages/learn/manage/account.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ title: Account management
 
 If you forget your password, you may request to reset it via the [password reset][password-reset] page. Enter the email address associated with your {{ $names.cloud.lower }} account. If the email address has an associated account, a password reset link will be sent to that address. Following the link, you will be able to enter a new password.
 
+## Add a password to social login
+
+To add a password to an account created with a social login (Google, Github), navigate to the *Preferences* page found by clicking on your profile in the top right of the dashboard. Under the *Account details* tab you can set a password for your account.
```
Comment (xginn8) [inline pages/learn/manage/account.md:17]: we should link to preferences here, just like in the following note

**Review by pdcastro — CHANGES_REQUESTED** [review]
Oh, I had missed Vipul's comment re commit message. There's a commit that reads literally:
 `Added 'a'`

**Comment by danthegoodman1** [comment]
> Oh, I had missed Vipul's comment re commit message. There's a commit that reads literally:
> `Added 'a'`

Yeah I did that because the first one I forgot to add a `Change-type`

**Comment by pdcastro** [comment]
> > Oh, I had missed Vipul's comment re commit message. There's a commit that reads literally:
> > `Added 'a'`
> 
> Yeah I did that because the first one I forgot to add a `Change-type`

The point is that `Added 'a'` means little when the sentence in read in the [CHANGELOG](https://github.com/balena-io/docs/blob/master/CHANGELOG.md), or with `git log`.

**Review by pdcastro — APPROVED** [review]

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by danthegoodman1** [comment]
@balena-ci rebase

**Review by pdcastro — CHANGES_REQUESTED** [review]
There is a merge commit in the list of commits. Please drop the merge commit, and use rebase instead of merge (we use rebase across all balena repos).

**Comment by danthegoodman1** [comment]
@balena-ci rebase

**Review by pdcastro — APPROVED** [review]

**Comment by pdcastro** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
Resolves #242

**Pull request merged**